packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -69,6 +69,10 @@ export async function generateCrudInput(
             decorators.push("@IsNumber()");
             decorators.push(`@Field(${prop.nullable ? "{ nullable: true }" : ""})`);
             type = "number";
+        } else if (prop.type === "number") {
```

Comment:
This could be moved into the else if branch above `else if (prop.type === "DecimalType" || prop.type === "number")`